components/sections/enchant-table.module.css

```
@@ -0,0 +1,492 @@
+  50% { opacity: 1; transform: scale(1.1) rotate(20deg); }
+}
+
+.ctaHint {
+  font-family: var(--font-display);
+  font-size: 11px;
+  letter-spacing: 2px;
+  color: var(--purple-1);
+  animation: blinkHint 1.6s steps(2) infinite;
+}
+
+@keyframes blinkHint {
```

Stylelint (17.14.1)

[error] Expected keyframe name "blinkHint" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)